vernac/vernacentries.ml

```diff
@@ -2218,13 +2219,15 @@ let translate_vernac ~atts v = let open Vernacextend in match v with
         Feedback.msg_notice @@ vernac_global_check c)
   | VernacPrint p ->
     VtReadProofOpt(fun ~pstate ->
-        Feedback.msg_notice @@ vernac_print ~pstate ~atts p)
+        unsupported_attributes atts;
+        Feedback.msg_notice @@ vernac_print ~pstate p)
   | VernacSearch (s,g,r) ->
     VtReadProofOpt(
         unsupported_attributes atts;
         vernac_search ~atts s g r)
-  | VernacLocate l -> unsupported_attributes atts;
+  | VernacLocate l ->
     VtReadProofOpt(fun ~pstate ->
+        unsupported_attributes atts;
         Feedback.msg_notice @@ vernac_locate ~pstate l)
```

Comment:
What does it change to have the attribute error inside or outside `VtReadProofOpt`?

Reply:
It delays the check to actual vernac execution vs the translation phase.

Indeed the translation function is not supposed to produce any side-effects as it could be useful for example for static analysis, so this is a micro bug fix.